components/reassign-user.tsx

```
@@ -0,0 +1,117 @@
+      onOpenChange(false)
+      setSelectedUser(null)
+    },
+    onError: () => {
+      toast.error('Failed to reassign task, please try again')
+    },
+  })
+
+  const handleReassign = () => {
+    if (!selectedUser) return
+
+    debugger
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)